schulcloud/News+CoreDataClass.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  News+CoreDataClass.swift
+//  
+//
+//  Created by Florian Morel on 03.01.18.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(News)
+public class News: NSManagedObject {
```

Comment:
Can you please rename this to `NewsArticle` to avoid confusion whether is object hold one or multiple news item.